### ISSUE TITLE
error when pip install multiPrime==2.4.7

### ISSUE THREAD

**Issue by wangzhichao1990** [joybio/multiPrime#6]
Hi,
When I installed this package, an error occurred.
![image](https://github.com/joybio/multiPrime/assets/14901250/2b5eefc4-82b6-42d9-82d5-5285559aa593)
Python and operating system information are shown in the following figure:
![image](https://github.com/joybio/multiPrime/assets/14901250/aff4be1f-81c6-4cb3-b93b-d08025549d2d)
Do you have any suggestions? thanks.

**Comment by joybio** [comment]
PyPI package support is currently unavailable due to a major update. I will address this issue in the near future. Meanwhile, you can install multiPrime using the following Conda command, which contains a complete pipeline:
```
conda create -n multiPrime -c bioconda -c conda-forge --file requirement2.txt
```

The PyPI package for multiPrime only includes the core primer design module. If you're interested, you can access our web-based version of multiPrime at this URL: http://123.60.87.239/MultiPrime/. While it's currently a work in progress, it is already functional and can effectively design primers.

Alternatively, you can download our GUI-based multiPrime2 for use on Windows systems from this GitHub repository: https://github.com/joybio/multiPrime2-GUI. 

Furthermore, both the web-based and GUI-based versions not only encompass the multiPrime primer design module but also house an enhanced version, multiPrime2, which is presently in the testing phase.

**Comment by wangzhichao1990** [comment]
The pipeline works well. Thank you very much for your development work.
By the way, I have some candidate primers for each pathogen. Is it possible to use the filtering algorithm in the package to find the optimal primer pool?

**Comment by joybio** [comment]
Certainly! You can locate the command line within the "multiPrime.py" file, just ensure that your input file is correctly specified.
